src/views/Lightning.vue

```diff
@@ -192,10 +192,16 @@
           </template>
           <template v-slot:menu>
             <b-dropdown-item
+              v-if="0 < numChannels"
               href="#"
               @click.stop.prevent="downloadChannelBackup"
               >Download Channel Backup</b-dropdown-item
             >
+            <b-dropdown-item
```

Comment:
Because there's already a "+ Open Chanel" button that's visible at all times, we shouldn't add it at two places. It would be a good idea to hide the entire meatballs menu if there are no channels since "Download Channel Backup" is the only menu option.

Reply:
That was my first thought but didn't want to be too heavy-handed. Changing now.